django/db/backends/oracle/utils.py

```diff
@@ -10,16 +10,21 @@ class InsertVar:
     insert statement.
     """
     types = {
+        'AutoField': int,
+        'BigAutoField': int,
```

Comment:
Please also add the new `SmallIntegerField` to this.

Reply:
and `SmallAutoField`

Reply:
I must have been half asleep when reviewing - sorry! I thought I had said `SmallAutoField`, but I was still right in suggesting the other 😅